dev.sh

```diff
@@ -13,4 +66,5 @@ docker run -it --rm --net host --privileged \
     -v /dev/shm:/dev/shm \
```

Comment:
Could you please add these volumes so Iceoryx inter process comunication is supported in the future?
```
-v /dev:/dev \
-v /tmp:/tmp \
```

Reply:
Done